### ISSUE TITLE
Faster rcnn nas loss doesn't go down

### ISSUE THREAD

**Issue by etatbak** [tensorflow/models#7849]
Hi, I am training custom objects on faster_rcnn_nas but total loss doesn't go down 0.9 or takes so long. I trained same dataset with faster_rcnn_inception_resnet_v2 loss goes under 0.1 around a couple hours.

Should i train more?

Here my tensorboard loss graph (faster_rcnn_nas):
![tb_nas](https://user-images.githubusercontent.com/46399056/69322087-852f4b00-0c44-11ea-9263-d01e874075dc.png)

------------------------

### System information
- **TensorFlow version: 1.14
- **GPU model and memory**: Nvidia quadro p5000

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
Bazel version
CUDA/cuDNN version
Exact command to reproduce

**Comment by pkulzc** [comment]
It depends on the loss weight setup. You should also look at mAP.

**Comment by etatbak** [comment]
> It depends on the loss weight setup. You should also look at mAP.

Detection is fine when i compare with other models. But only the problem is high loss value while training. Also a couple false negatives- not a big issue. I didn't change loss weight from config file.

**Comment by anacondabitch** [comment]
same problem here ... true positive detection is awesome bat i havee reverse problem ...so many false positive in evaluation ???

**Comment by gorkem7** [comment]
Same here, I am at ~90K steps at the moment with 4GPU (4x Tesla V100 32GB). Detection quality is way better than Faster_RCNN_101 but loss value does not under 0.85
